lightning/src/ln/channelmanager.rs

```diff
@@ -5622,6 +5622,12 @@ where
 				Some(inbound_chan) => {
 					match inbound_chan.funding_created(msg, best_block, &self.signer_provider, &self.logger) {
 						Ok(res) => res,
+						Err((inbound_chan, ChannelError::Ignore(_))) => {
+							// If we get an `Ignore` error then something transient went wrong. Put the channel
+							// back into the table and bail.
```

Comment:
when does this get retried (would be good to doc here)?

Reply:
I am assuming that there will be a to-be-determined mechanism that allows the remote signer's implementation to initiate a restart.